main.css

```diff
@@ -0,0 +1,746 @@
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+}
+
+.artists-section-heading {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.artists-heading {
+  text-align: center;
+  font-size: 30px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: -0.01em;
+}
+
+.artists-div-bar {
+  width: 53px;
+  height: 2px;
+  border: none;
+  background: #ef513c;
+  margin: 10px auto;
+}
+
+.artists-grid {
+  display: grid;
+  grid-template-columns: 100%;
+}
+
+.artist {
+  display: grid;
+  gap: 6%;
+  grid-template-columns: 34% 60%;
+  margin: 12px;
+}
+
+.artist-img {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+
+.artist-img img {
+  width: 100%;
+}
+
+.artist-description {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+
+.artist-name {
+  padding-top: 10px;
+  font-size: 15px;
+  font-weight: 700;
+  line-height: 18px;
+  letter-spacing: -0.02em;
+  text-align: left;
+}
+
+.artist-position {
+  font-size: 10px;
+  font-weight: 700;
+  line-height: 15px;
+  letter-spacing: -0.02em;
+  color: var(--orange-shade-color);
+}
+
+.artisttile-div-bar {
+  width: 30px;
+  height: 2px;
+  background: var(--gray-shade-color);
+}
+
+.artist-info {
+  font-size: 9px;
+  font-weight: 700;
+  line-height: 14px;
+  letter-spacing: -0.06em;
+}
+
+.moreButton {
+  width: 100%;
+  height: 60px;
+  border: 5px solid var(--orange-shade-color);
+  color: #000;
+  background: none;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  font-size: 14px;
+}
+
+.dropdown {
+  color: var(--orange-shade-color);
+  font-size: 20px;
+}
+
+.mobile-tiles-display {
+  display: none;
+}
+
+/* Transitions */
+.cards:hover {
+  border: 2px solid var(--gray-shade-color);
+  border-radius: 2px;
+  box-shadow: 0 4px 18px 0 rgba(233, 200, 200, 0.25);
+  transform: scale(1.06);
+}
+
+.nav-list-items li:hover {
+  transform: scale(1.2) translateY(6px);
+  box-shadow: 0 4px 18px 0 rgba(216, 113, 113, 0.25);
+}
+
+.wos-tour-button:hover {
+  background: var(--orange-shade-color);
+  color: #fff;
+}
+
+@keyframes shake {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  25% {
+    transform: rotate(5deg);
+  }
+
+  50% {
+    transform: rotate(0deg);
+  }
+
+  75% {
+    transform: rotate(-5deg);
+  }
+
+  100% {
+    transform: rotate(0deg);
+  }
+}
+
+.logo-img-link:hover {
+  animation-name: shake;
+  -webkit-animation-name: shake;
+  animation-duration: 0.35s;
+  animation-timing-function: ease-in;
+}
+
+@media (min-width: 768px) {
+  header {
+    height: 105px;
+  }
+
+  main {
+    top: 122px;
+    width: 100%;
+  }
+
+  .desktop-nav {
+    display: block;
+    width: 100%;
+    height: 100%;
+  }
+
+  .header-bar {
+    width: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    display: flex;
+    justify-content: flex-end;
+    background: #3e3e3e;
+  }
+
+  .header-bar-list {
+    display: flex;
+    gap: 24px;
+    list-style: none;
+    color: #fff;
+    margin: 6px;
+  }
```

Comment:
- [ ] Please add spacing to the right of this div to match the given design layout:
yours:
![image](https://github.com/tan12082001/Project-Capstone-1/assets/91486791/f8d8720e-7357-4f5f-8009-f88b0cd6c897)
given: 
![image](https://github.com/tan12082001/Project-Capstone-1/assets/91486791/db4738b8-fe53-469b-9ac3-bbc1e74484fc)